src/rustllvm/PassWrapper.cpp

```diff
@@ -101,11 +101,13 @@ extern "C" LLVMPassRef LLVMRustCreateModuleAddressSanitizerPass(bool Recover) {
 }
 
 extern "C" LLVMPassRef LLVMRustCreateMemorySanitizerPass(int TrackOrigins, bool Recover) {
-#if LLVM_VERSION_GE(8, 0)
+#if LLVM_VERSION_GE(9, 0)
   const bool CompileKernel = false;
 
   return wrap(createMemorySanitizerLegacyPassPass(
       MemorySanitizerOptions{TrackOrigins, Recover, CompileKernel}));
+#elif LLVM_VERSION_GE(8, 0)
+  return wrap(createMemorySanitizerLegacyPassPass(TrackOrigins, Recover));
```

Comment:
This is the LLVM 8 fix.

cc @rust-lang/infra -- do we have any budget for more CI builders? Currently we only test the _minimum_ external LLVM and rust's own src/llvm-project fork, but there are more possible LLVM versions between...

Reply:
I suspect no, or if we did, this would not be a good use IMO.

Reply:
Even just `./x.py check` on those middle versions would be an improvement.

Reply:
Currently x.py check doesn't do anything at all LLVM-wise -- we don't attempt to compile it, intentionally, so that even people with smaller machines can at least sort of contribute to the compiler.

Reply:
Ah, darn. We _could_ let that build script work in full when using external LLVM, which shouldn't be expensive, but I won't press further on this point. Thanks anyway.